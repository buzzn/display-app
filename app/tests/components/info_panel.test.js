import React from 'react';
import renderer from 'react-test-renderer';
import InfoPanel from '../../components/info_panel';

describe('info panel component tests', () => {
  test('renders in text correctly', () => {
    const tree = renderer.create(
      <InfoPanel type="in"/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('renders out text correctly', () => {
    const tree = renderer.create(
      <InfoPanel type="out"/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('renders data correctly', () => {
    const tree = renderer.create(
      <InfoPanel data={ 10000 }/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
