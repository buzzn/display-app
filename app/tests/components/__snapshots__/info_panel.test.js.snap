// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`info panel component tests renders data correctly 1`] = `
<div>
  Tagesproduktion 
  <span
    style={
      Object {
        "fontWeight": "bold",
      }
    }
  >
    10 kWh
  </span>
</div>
`;

exports[`info panel component tests renders in text correctly 1`] = `
<div>
  Tagesverbrauch 
  <span
    style={
      Object {
        "fontWeight": "bold",
      }
    }
  >
    n.a.
  </span>
</div>
`;

exports[`info panel component tests renders out text correctly 1`] = `
<div>
  Tagesproduktion 
  <span
    style={
      Object {
        "fontWeight": "bold",
      }
    }
  >
    n.a.
  </span>
</div>
`;
